lib/Data/Validate/Sanctions.pm

```diff
@@ -57,13 +57,15 @@ sub is_sanctioned {        ## no critic (RequireArgUnpacking)
 
     for my $k (sort keys %$data) {
         foreach my $name (@{$data->{$k}{names}}) {
+            (my $check_name = $name) =~ s/[[:^alpha:]]//g;
+            $check_name = uc($check_name);
             for (@name_variants) {
-                return $k if $name =~ /$_/;
-        }
+                return wantarray ? ($k, $name) : $k if $check_name =~ /$_/;
```

Comment:
worth a `/\Q$_/` here to make the intent slightly clearer? (I don't think we'd expect to have any regex characters at that point, so it shouldn't affect the functionality)